Add contact form for testing

eirizarry opened the pull request:
Closes #115

I have added a simple html form and php script to test using an email contact form on our website. The email should return the user's name, email, and message, sent from the root in our email server.
This file was placed in a temporary folder.

If further testing is needed, results will be documented in issue 115.

eirizarry committed:
Add contact form to "Contact" page

- This replaces the previous content for this page with a contact form

eirizarry committed:
Restore contact page to original content

- Was testing this for a php email script, but have decided to use a different page to test this function in production

eirizarry committed:
Add test php script and form

eirizarry committed:
Rename email recipient and add user email

- These files have been moved to a 'temp' folder for testing

eirizarry committed:
Merge branch 'develop' into feature/contact-page

Commit:
Merge branch 'develop' into feature/contact-page

pdavidc reviewed (changes requested):
The /temp folder does not appear in the Hugo build output. Doesn't web content need to be in /content or /static?

eirizarry commented:
I will move it, and see if that works.

eirizarry committed:
Move contact form to 'content' folder

-Move html form and php script to content folder
-Generate index.html as a .md file

eirizarry committed:
Reformat contact-form

Reformat html so that form displays properly on the page

eirizarry committed:
Rename contactFormEmail.php

Hugo does not build a page name with camelCasing.

pdavidc reviewed (approved):
Good luck with the test!
